src/app/components/audios/audios.component.html

```diff
@@ -1 +1,65 @@
-<h3 class="govuk-heading-s">Your Audio</h3>
+<ng-template #tableTemplatePartial let-row>
+  <td class="govuk-table__cell">
+    <a [routerLink]="['.']"> {{ row.caseNumber }}</a>
```

Comment:
Should link to case screen?

Reply:
The current response doesn't include a case_id only a case_number therefore we are unable to link back currently. I did mention this too @hemantasharma1129 and he raised a ticket I think.

Reply:
Noted.